notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -61,7 +61,7 @@
     "%env BUCKET=$BUCKET\n",
```

Comment:
<p>I suggest we remove this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Or, we can add a header which says that this section is optional. It is an optional part and nice to show an example on how it is done in python SDK which is more readable </p>

Reply:
<p>How about moving this  [Raw cell] cell to markdown, add Python syntax highlight?</p><p>I believe that is much more readable and useful than plain python text.</p><p> </p><p>So far I push the code with it, so please let me know what you think.</p>